.github/workflows/optional-test.yaml

```diff
@@ -108,7 +111,6 @@ jobs:
     runs-on: ubuntu-20.04
 
     env:
-      SKIP_PRISMA_VERSION_CHECK: true # see https://github.com/MichalLytek/typegraphql-prisma/issues/31
```

Comment:
I think that is still needed, as otherwise the check will fail for some versions until typegraphql-prisma is updated.

Reply:
I moved it into the script directly as it was not obvious to me when running locally

Reply:
Good point!